modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/published_document_success_message.jspf

```diff
@@ -0,0 +1,23 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%
+String publishedDocumentMessage = GetterUtil.getString(SessionMessages.get(request, "publishedDocumentRequestProcessedSuccess"));
+%>
+
+<c:if test="<%= Validator.isNotNull(publishedDocumentMessage) %>">
```

Comment:
I've seen that this jspf is consistent with friendly_url_changed_message.jspf but the code is different.

In this pages we use:

```
<aui:script>
		Liferay.Util.openToast({
			message: '<%= friendlyURLChangedMessage %>',
			toastProps: {
				autoClose: 20000,
			},
			type: 'warning',
		});
	</aui:script>
```
It could be nice if we make this logic consisten in both cases.

Reply:
honestly, I do not know what is the best way to put this. 
So I take the last development way (KB article) to do it but I do not know if this is the way or not

@ambrinchaudhary  what is the better way to implement this?

Reply:
Actually, the best way is the current approach, we should try to avoid calling global methods (Liferay.Util.openToast) but directly use the frontend component. 
This was discussed a month ago in frontend slack channels, I am sorry I cannot find it now.

Reply:
Found the reference to slack thread: https://liferay.slack.com/archives/CNBG06JS3/p1687448868326979

Reply:
so the best way is the aui:script ?